evaluation/src/main/java/com/formulasearchengine/mlp/evaluation/EvaluateCommand.java

```diff
@@ -16,19 +16,28 @@
         "\n1,matched,\"W\",\"van der waerden number\"")
   private String in = "";
 
-  public String getGold() {
-    return gold;
-  }
 
   @Parameter(names = {"-gold"},
     description = "The gold standard file")
   private String gold = "";
 
+  @Parameter(names = {"-titleKey"},
+    description = "Use the title instead of the qId for the matching")
```

Comment:
The idea of the -titleKey option is not yet clear to me. Maybe you can a few sentences here to explain why one would need such a feature?